src/utils/headers.js

```diff
@@ -1,6 +1,7 @@
 const { parseAllHeaders } = require('netlify-headers-parser')
 
-const { NETLIFYDEVWARN } = require('./logo')
+const { log } = require('./command-helpers')
```

Comment:
@tinfoil-knight, you changes are helping us to write cleaner code ❤️

Reply:
Thanks @tinfoil-knight!

Reply:
Thank you both for making netlify a good home for all contributors.